bugbug/bugzilla.py

```diff
@@ -39,6 +39,7 @@
     "Firefox Health Report",
     # 'Focus',
     # 'Hello (Loop)',
+    "Invalid Bugs::General",
```

Comment:
Are you sure this is working?
This is a list of products, the product should be just "Invalid Bugs" ("General" is a component).

Reply:
Ah, sorry forgot to update this.